cmd/completion/completion.go

```diff
@@ -78,13 +73,17 @@ func RunCompletion(out io.Writer, cmd *cobra.Command, args []string) {
 	fmt.Printf("Unsupported shell type %q.\n", args[0])
 }
 
-//RunCompletionBash is used for the bash shell
+// RunCompletionBash is used for the bash shell
 func RunCompletionBash(out io.Writer, cmd *cobra.Command) {
-	cmd.GenBashCompletion(out)
+	err := cmd.GenBashCompletion(out)
+	if err != nil {
+		klog.Error(err)
+	}
 }
 
-//RunCompletionZsh is used for the zsh shell
+// RunCompletionZsh is used for the zsh shell
 func RunCompletionZsh(out io.Writer, cmd *cobra.Command) {
```

Comment:
Can we use, `cmd.GenZshCompletion(out)` in line:70, this would do the job of `RunCompletionBash`